json_test.go

```diff
@@ -6,11 +6,10 @@ import (
 
 	"github.com/ncruces/go-sqlite3"
 	_ "github.com/ncruces/go-sqlite3/embed"
-	_ "github.com/ncruces/go-sqlite3/vfs/memdb"
 )
 
 func Example_json() {
-	db, err := sqlite3.Open("file:/test.db?vfs=memdb")
+	db, err := sqlite3.Open(":memory:")
```

Comment:
You switched this one to `:memory:` but the JSON example in driver still uses memdb. Should that one switch too?

Reply:
This is for "easy" (or rather "rule-of-thumb") consistency.

I used:
- `:memory:` for the low level `sqlite3` bindings, where it's usually fine (unless I'm testing `memdb` specifically)
- `memdb` for `database/sql` (even when `:memory:` would be fine, because no data ever is stored)

This reduces cognitive load on me (and hopefully users).

Reply:
To be clearer: most tests want a throwaway isolated database **and** don't open multiple connections, so `:memory:` **is** fine.

It's the `database/sql` connection pool under the covers that causes issues (and even then, mostly in the form of "this works 95% of the time" flakiness).

Reply:
Also, another reason, on testable examples the new `memdb.TestDB(t)` doesn't work (because we don't have the `t`).

So options are:
1. plain old file (needs to be deleted)
2. less ideal "shared database that dies if ref-count reaches 0" (works 99.999% but is theoretically wrong)
3. `:memory:` (goes against "rule-of-thumb")

Trying to keep examples straightforward, I've used a mix of 1 and 2.